### PR TITLE
feat: activate not-exists for link prop

### PR DESCRIPTION
**Pull request by Vijeinath**
resolves DEV-1966

### CODE REVIEW

File: apps/dsp-app/src/app/workspace/search/advanced-search/resource-and-property-selection/search-select-property/specify-property-value/specify-property-value.component.spec.ts [hunk 1]
```diff
@@ -323,45 +323,45 @@ describe('SpecifyPropertyValueComponent', () => {
         ).toEqual(8);
     });
 
-    // it('should set the correct comparison operators for a linking property type (on top level)', () => {
```
Comment (EricSommerhalder) [inline apps/dsp-app/src/app/workspace/search/advanced-search/resource-and-property-selection/search-select-property/specify-property-value/specify-property-value.component.spec.ts:326]: Is this dead code now? I'd delete it if it's not used.

File: apps/dsp-app/src/app/workspace/search/services/gravsearch-generation.service.spec.ts [hunk 2]
```diff
@@ -2246,54 +2246,54 @@ OFFSET 0
         ).toHaveBeenCalledTimes(1);
     });
 
-//     it('Gravsearch query string with no existing LINK property values', () => {
```
Comment (EricSommerhalder) [inline apps/dsp-app/src/app/workspace/search/services/gravsearch-generation.service.spec.ts:2249]: Same as above

**Review by EricSommerhalder — APPROVED** [review]
LGTM, minor comments

File: apps/dsp-app/src/app/workspace/search/advanced-search/resource-and-property-selection/search-select-property/specify-property-value/specify-property-value.component.spec.ts (hunk 1 above)
Comment (Vijeinath) [reply apps/dsp-app/src/app/workspace/search/advanced-search/resource-and-property-selection/search-select-property/specify-property-value/specify-property-value.component.spec.ts:326]: No, I uncommented it so the test is valid for t

**Pull request merged**